### DISCUSSION TITLE
Auto refresh blocker?

### DISCUSSION THREAD

**Discussion by casimoes03** [bromite/bromite#1115] [Ideas]
How about adding this enhancement to Bromite on a per-site basis? There are some sites that frequently auto refreshes the page and then, somehow, the point where the page was before the refresh is changed, as www.folha.com.br for example. It is very annoying. This is easily observed from the position of the  middle to end of the page and does not happens at start of it.
As this happens with chromium too, I assume that this behavior is a result of a bug in the site or in the browser upstream and the best solution to this problem is to disable auto refresh on a per-site basis.
There is an extension for chrome named Auto Refresh Blocker that do exactly that.

**Comment by csagan5** [comment]
You can disable Javascript for this website.

**Comment by casimoes03** [comment]
When JavaScript is disabled in the home page, part of the content is missed. I already disabled JavaScript for other pages in this site, but I really would not like to disable it in the home page.

I think that very few people will use a auto refresh blocker, but, as I said, this feature could mask a bug in the wrong positioning of the page when returning to it from another page and also reduce the loading time.

Anyway thanks for the suggestion, csagan5
